drivers/i2c/Kconfig.sbcon

```diff
@@ -6,7 +6,7 @@
 
 menuconfig I2C_SBCON
 	bool "I2C driver for ARM's SBCon two-wire serial bus interface"
-	depends on I2C
+	depends on I2C && ARM
```

Comment:
Why add ARM?  Just don't include it on the board config, right?

Reply:
Oops. This should be in another pull request.